src/database/migration/file/migration_version_8_users_add_anon_user_info_emoji_name.ts

```diff
@@ -0,0 +1,27 @@
+import {SQLiteDatabase} from 'react-native-sqlite-storage';
+
+import Migration from './migration.types';
+import UserSchema from '../../schema/UserSchema';
+
+const TABLE_NAME = UserSchema.getTableName();
+const COLUMN_NAME_ADDED = 'anon_user_info_emoji_name';
+
+class MigrationVersion8 implements Migration {
+  up = async (db: SQLiteDatabase): Promise<void> => {
+    const upQuery = `ALTER TABLE ${TABLE_NAME} ADD COLUMN ${COLUMN_NAME_ADDED} TEXT;`;
+
+    console.log(`====== MIGRATING: ${TABLE_NAME.toLocaleUpperCase()} TABLE ======`);
+
+    await db.executeSql(upQuery);
+    console.log(`====== DONE MIGRATING: ${TABLE_NAME.toLocaleUpperCase()}  TABLE =====`);
+  };
+
+  down = async (db: SQLiteDatabase): Promise<void> => {
+    console.log(`====== DOWNGRADING: ${TABLE_NAME.toLocaleUpperCase()} TABLE ======`);
+    const downQuery = `ALTER TABLE ${TABLE_NAME} DROP COLUMN ${COLUMN_NAME_ADDED};`;
+    await db.executeSql(downQuery);
+    console.log(`====== DONE DOWNGRADING: ${TABLE_NAME.toLocaleUpperCase()}  TABLE ======`);
+  };
+}
+
+export default new MigrationVersion8();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code looks good overall, but there's a potential issue with the `down` method. SQLite does not support dropping a column directly using the `ALTER TABLE DROP COLUMN` syntax. You might need to create a new table without this column and copy the data over.

```diff
  down = async (db: SQLiteDatabase): Promise<void> => {
    console.log(`====== DOWNGRADING: ${TABLE_NAME.toLocaleUpperCase()} TABLE ======`);
- 21:     const downQuery = `ALTER TABLE ${TABLE_NAME} DROP COLUMN ${COLUMN_NAME_ADDED};`;
+ 21:     // SQLite doesn't support dropping columns. You'll need to create a new table without this column and copy the data over.
    await db.executeSql(downQuery);
    console.log(`====== DONE DOWNGRADING: ${TABLE_NAME.toLocaleUpperCase()}  TABLE ======`);
  };
```
Please consider revising your downgrade migration strategy for SQLite.


<!-- This is an auto-generated comment by OSS CodeRabbit -->